tests/test_core/test_decorator_calls.py

```diff
@@ -25,33 +33,37 @@ def test_basic_decorator():
     list(conn.get_all_instances()).should.equal([])
 
 
-def test_context_manager():
+def test_context_manager(*args):
     conn = boto.connect_ec2('the_key', 'the_secret')
-    with assert_raises(EC2ResponseError):
-        conn.get_all_instances()
+    with mock.patch('boto.connection.AWSQueryConnection.make_request', return_value=MockHttpResponse()):
```

Comment:
I tested again and with Python 3.4, it block the test suite without end...

Tests doesn't respond to `KeyboardInterrupt` and I must `kill` the process.
So I mock out the blocking line.